src/ontology/dpo.Makefile

```diff
@@ -34,6 +28,47 @@ tmp/remaining_classes.txt: tmp/all_patternised_classes.txt tmp/all_defined_class
 tmp/remaining_definitions.owl: $(SRC) tmp/remaining_classes.txt
 	$(ROBOT) filter -i $< -T tmp/remaining_classes.txt --axioms "equivalent annotation" --trim false -o $@
 
+##################################
+##### Custom mirroring rules #####
+##################################
+
+# FBdv defines "shorthands" for some RO relations, which leads to
+# "duplicate label" violations. In theory we could use 'make_base' to
+# automatically get rid of those duplicated labels, but doing so has the
+# the side-effect of removing the "Transitive" characteristic on
+# FBdv:0018001, which in turn prevents the correct classification of
+# lethal terms.
+# So here, we take a softer approach, in which we simply remove from
+# FBdv's base the annotations on the RO relations for which we have
+# defined shorthands.
+# (Whether it is a good idea to have those shorthands defined in FBdv
+#  in the first place is another question.)
+mirror-fbdv: | $(TMPDIR)
+	if [ $(MIR) = true ] && [ $(IMP) = true ]; then \
+		curl -L $(OBOBASE)/fbdv/fbdv-base.owl --create-dirs \
+		  -o $(TMPDIR)/fbdv.owl --retry 4 --max-time 200 && \
+		$(ROBOT) remove -i $(TMPDIR)/fbdv.owl \
+		  --term RO:0002012 --term RO:0002087 --term RO:0002090 \
+		  --axioms annotation \
+		  convert -o $@.tmp.owl && mv $@.tmp.owl $(TMPDIR)/$@.owl; fi
+
+# RO's base contains duplicated definitions for several BFO classes that
+# presumably do *not* belong in RO. And here, using 'make_base' is not
```

Comment:
But it is better to "use_base", I agree

Reply:
You mean doing something like this:

```
- id: ro
  make_base: TRUE
  base_iris:
    - http://purl.obolibrary.org/obo/RO_
    - http://purl.obolibrary.org/obo/BFO_0000050
    - http://purl.obolibrary.org/obo/BFO_0000051
    - http://purl.obolibrary.org/obo/BFO_0000063
    ...
```

and so on for all BFO-style IRIs that actually belong to RO?

I’ll admit it’s somewhat cleaner (though I don’t like the fact that one would need to run `update_repo` just to add another RO-owned BFO relation here, if you realise you have missed one), but it’s inoperant due to the `robot remove` bug I mentioned above: using `make_base` here would lead to the loss of all transitivity characteristics in all RO relations.

Reply:
yes, this makes all sense.